tests/lib/rules/no-empty-function.js

```diff
@@ -267,6 +269,16 @@ ruleTester.run("no-empty-function", rule, [
         {
             code: "var foo = () => 0;",
             parserOptions: { ecmaVersion: 6 }
+        },
+        {
+            code: "const foo = { async method() {} }",
```

Comment:
Can we add more tests for this change? It doesn't look like we're adding any new invalid tests right now.